src/BloomBrowserUI/bookEdit/css/editMode.css

```diff
@@ -213,7 +213,7 @@ TEXTAREA.province {
 .overflow p:empty:after {
   content: '¶';
 }
-.marginOverflow {
+.childOverflowingThis {
   border-bottom: solid thick red !important;
   color: red !important;
```

Comment:
This will turn the whole box with the overflowing child red? All of its text?

Also I'm surprised not to see anything done when thisOverflowingParent.